ambry-messageformat/src/test/java/com.github.ambry.messageformat/BlobStoreHardDeleteTest.java

```diff
@@ -138,12 +138,15 @@
     private MessageFormatInputStream getPutMessage(StoreKey key, BlobProperties blobProperties, byte[] usermetadata,
         byte[] blob, int blobSize, short blobVersion, BlobType blobType)
         throws MessageFormatException {
-      if (blobVersion == MessageFormatRecord.Blob_Version_V1) {
-        return new PutMessageFormatInputStream(key, blobProperties, ByteBuffer.wrap(usermetadata),
-            new ByteBufferInputStream(ByteBuffer.wrap(blob)), blobSize, blobType);
-      } else {
-        return new PutMessageFormatBlobV2InputStream(key, blobProperties, ByteBuffer.wrap(usermetadata),
-            new ByteBufferInputStream(ByteBuffer.wrap(blob)), blobSize, blobType);
+      switch (blobVersion) {
```

Comment:
This is no different from the other cases. Not a big deal, but let us just be consistent and use the ternary operator everywhere if that is what you prefer.